core/src/main/scala/org/dbpedia/spotlight/db/memory/MemoryVectorStore.scala

```diff
@@ -33,16 +32,18 @@ class MemoryVectorStore extends MemoryStore with KryoSerializable{
   }
 
   def onNilIndex(string: String) = {
-    SpotlightLog.warn(this.getClass, "Warning: token " + string + " not in dictionary! Lookup returning null vector.")
```

Comment:
It will log quite a lot, since quite a lot of tokens are not in the dictionary. While annoying, it would probably be fair to point that out. Maybe debug log level rather than warn?